src/connectors/index.ts

```diff
@@ -29,9 +29,9 @@ export function getNetworkLibrary(): BaseProvider {
 
 function createInjectedMetaMaskProvider() {
   return new InjectedConnector({
-    // mainnet, ropsten, rinkeby, goerli, optimism, kovan, fantom, local buidler
+    // mainnet, ropsten, rinkeby, goerli, optimism, kovan, evmos, fantom, local buidler
     // see: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md
-    supportedChainIds: [1, 3, 4, 5, 10, 42, 250, 42161, 31337],
+    supportedChainIds: [1, 3, 4, 5, 10, 42, 250, 9000, 9001, 42161, 31337],
```

Comment:
we could probably derive this from SUPPORTED_NETWORKS

Reply:
When I try looping over SUPPORTED_NETWORK keys using Object.keys(SUPPORTED_NETWORK) I get a webpack related circular dependency issue.

![image](https://user-images.githubusercontent.com/5507707/164794199-80f7f5a8-7c1c-4334-b19f-2ade912737f1.png)